judgels-frontends/raphael/src/components/MathjaxWrapper/MathjaxWrapper.tsx

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
+import { HtmlText } from '../HtmlText/HtmlText';
 
 declare global {
   interface Window {
     MathJax: any;
   }
 }
 
-export class MathjaxWrapper extends React.Component {
+interface MathjaxProps {
+  text: string;
+}
+export class MathjaxWrapper extends React.Component<MathjaxProps> {
+  isContainMathJax = false;
+
+  constructor(props) {
```

Comment:
This kind of derived props is conventionally implemented as state in React.

E.g. should be something like:

```
constructor(props) {
  super(props);
  this.state = {
    containsMathJax: this.containsMathjaxSyntax(props.children),
  };
}
```

Then later it is referenced as `this.state.containsMathJax`

Reply:
Changed.